org.eclipse.jdt.core.tests.compiler/src/org/eclipse/jdt/core/tests/compiler/regression/Deprecated9Test.java

```diff
@@ -915,7 +915,15 @@ public void testBug533063_2() throws Exception {
 			"----------\n";
 		runner.runWarningTest();
 	}
-	public void testBug534304() throws Exception {
+	public void testBug534304_1() throws Exception {
+		Map<String, String> options= getCompilerOptions();
+		String compliance= options.get(CompilerOptions.OPTION_Compliance);
+		if (compliance.equals(CompilerOptions.VERSION_9) ||
```

Comment:
In compiler tests we typically check compliance like this:
`this.complianceLevel < ClassFileConstants.JDK13`

Reply:
Changed after rebasing.